### DISCUSSION TITLE
A few errors in the email widget

### DISCUSSION THREAD

**Discussion by Hamster-of-power** [streetturtle/awesome-wm-widgets#358] [General]
Hi,
I am new to Github, so sorry, if this is not the "correct" way of reporting. There were a few errors that I would like to point out:

1. It says
email_icon:set_image(path_to_icons .. "/mail-mark-new.png")
That, png does not exist. Maybe try
email_icon:set_image(path_to_icons .. "/mail-message-new.png")?

2. In the show_emails()function, you are using python. However, it requires python3. For python3, however, the print function of
/home/<username>/.config/awesome/email-widget/read_unread_emails.py
needs brackets in two places:
line 10
     print("No messages found!")
and line 16
    print("ERROR getting message", num)

Great program btw.

**Comment by streetturtle** [comment]
Thanks for the feedback! I created an issue from this post: https://github.com/streetturtle/awesome-wm-widgets/issues/369
